src/test/java/teammates/test/cases/webapi/UpdateInstructorActionTest.java

```diff
@@ -0,0 +1,201 @@
+package teammates.test.cases.webapi;
+
+import org.apache.http.HttpStatus;
+import org.testng.annotations.Test;
+
+import teammates.common.datatransfer.attributes.InstructorAttributes;
+import teammates.common.exception.InvalidHttpRequestBodyException;
+import teammates.common.exception.NullHttpParameterException;
+import teammates.common.util.Const;
+import teammates.common.util.FieldValidator;
+import teammates.logic.core.CoursesLogic;
+import teammates.logic.core.InstructorsLogic;
+import teammates.ui.webapi.action.JsonResult;
+import teammates.ui.webapi.action.UpdateInstructorAction;
+import teammates.ui.webapi.output.MessageOutput;
+import teammates.ui.webapi.request.InstructorCreateRequest;
+
+/**
+ * SUT: {@link UpdateInstructorAction}.
+ */
+public class UpdateInstructorActionTest extends BaseActionTest<UpdateInstructorAction> {
+
+    private final InstructorsLogic instructorsLogic = InstructorsLogic.inst();
+
+    @Override
+    protected String getActionUri() {
+        return Const.ResourceURIs.INSTRUCTORS;
+    }
+
+    @Override
+    protected String getRequestMethod() {
+        return PUT;
+    }
+
+    @Override
+    @Test
+    protected void testExecute() {
+        InstructorAttributes instructorToEdit = typicalBundle.instructors.get("instructorNotDisplayedToStudent1");
+        String instructorId = instructorToEdit.googleId;
+        String courseId = instructorToEdit.courseId;
+
+        loginAsInstructor(instructorId);
+
+        ______TS("Typical case: edit instructor successfully");
+
+        String[] submissionParams = new String[] {
+                Const.ParamsNames.COURSE_ID, courseId,
+        };
+
+        String newInstructorName = "newName";
+        String newInstructorEmail = "newEmail@email.com";
+
+        InstructorCreateRequest reqBody = new InstructorCreateRequest(instructorId, newInstructorName,
+                newInstructorEmail, instructorToEdit.role,
+                instructorToEdit.displayedName, instructorToEdit.isDisplayedToStudents);
+
+        UpdateInstructorAction updateInstructorAction = getAction(reqBody, submissionParams);
+        JsonResult actionOutput = getJsonResult(updateInstructorAction);
+
+        MessageOutput test = (MessageOutput) actionOutput.getOutput();
+        System.out.println(test.getMessage());
```

Comment:
I believe these two lines weren't meant to be committed?

Reply:
Oh sry! Will ensure this does not happen again